extensions/cornerstone/src/commandsModule.ts

```diff
@@ -322,15 +323,11 @@ function commandsModule({
         return;
       }
 
-      const filteredViewports = viewports.filter(viewport => {
-        if (!viewport.viewportOptions) {
-          return false;
+      const filteredViewports = Array.from(viewports.values()).filter(
+        viewport => {
+          return toolGroupViewportIds.includes(viewport.viewportId);
```

Comment:
Why do we no longer check `if(!viewport.viewportOptions)`?

Reply:
since previously viewportId was in the options and now it is not